docs/openapi/schemas.yaml

```diff
@@ -1598,3 +1598,381 @@ ApiKeyResponse:
+  description: |
+    Origin of opportunity.
+    - AUTOMATION: generated by automation
+    - ESS_OPS: generated manually by Experience Success Engineers
+    - AI: generated by AI
+  type: string
+  enum:
+    - 'AUTOMATION'
+    - 'ESS_OPS'
+    - 'AI'
+  example: 'ESS_OPS'
+Opportunity:
+  type: object
+  properties:
+    id:
+      description: UUID of this opportunity
+      $ref: '#/Id'
+    siteId:
+      description: UUID of the site this opportunity is for
+      $ref: '#/Id'
+    auditId:
+      description: UUID of the audit result that triggered this opportunity
+      $ref: '#/Id'
+    runbook:
+      description: URL to the runbook for this opportunity
+      type: string
+      format: url
+      example: 'https://example.com/runbook/broken-backlinks'
+    type:
+      $ref: '#/OpportunityType'
+    data:
+      description: Optional map of type-specific data relevant to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksOpportunityData'
+    origin:
+      $ref: '#/OpportunityOrigin'
+    title:
+      description: Title of this opportunity
+      type: string
+      example: 'Authoritative domains are linking to invalid SecurBank URLs. This could impact your SEO.'
+    description:
+      description: Description of this opportunity
+      type: string
+      example: 'Provide the correct target URL that each of the broken backlinks should be redirected to.'
+    status:
+      description: Status of this opportunity
+      $ref: '#/OpportunityStatus'
+    guidance:
+      description: Guidance text specific to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksGuidance'
+    tags:
+      type: array
+      items:
+        type: string
+      description: Optional list of tags
+      example:
+        - 'Traffic acquisition'
+    createdAt:
+      description: UTC timestamp of when the opportunity was created
+      $ref: '#/DateTime'
+    createdBy:
+      type: string
+      description: |
+        Initially not available.
+        
+        IMS or other user ID of who created this opportunity
+      example: '47995190583E98E70A495D53@adobe.com'
+    updatedAt:
+      description: UTC timestamp of when the opportunity was last updated
+      $ref: '#/DateTime'
+    updatedBy:
+      description: |
+        Initially not available.
+        
+        IMS or other user ID of who updated this opportunity
+      type: string
+      example: '67994690583E98E70A495D53@adobe.com'
+  required:
+    - id
+    - siteId
+    - auditId
+    - runbook
+    - type
+    - origin
+    - title
+    - status
+    - createdAt
+    - createdBy
+    - updatedAt
+    - updatedBy
+OpportunityList:
+  type: array
+  items:
+    $ref: '#/Opportunity'
+OpportunityCreate:
+  type: object
+  properties:
+    siteId:
+      description: UUID of the site this opportunity is for
+      $ref: '#/Id'
+    auditId:
```

Comment:
Don't know why, but `auditId` is missing from the docs.
Is it a request body parameter or a request path parameter?
![Screenshot 2024-11-15 at 16 51 06](https://github.com/user-attachments/assets/c958280e-4fab-406e-9f67-8c6b5689ad30)

Reply:
It's a request body parameter.
I will check on Monday where it got lost.

Reply:
Found. The referenced `Id` schema had `readOnly: true`, so the property wasn't rendered in request body. We needed to explicitly set `readOnly: false` for the property in the request body schema.